osbs/conf.py

```diff
@@ -296,6 +296,9 @@ def get_distribution_scope(self):
     def get_architecture(self):
         return self._get_value("arch", self.conf_section, "architecture")
 
+    def get_platform(self):
+        return self._get_value("platform", self.conf_section, "platform")
```

Comment:
Added so we can test worker build from CLI, perhaps a `--platform` argument is more desirable?